CHANGELOG.md

```diff
@@ -7,6 +7,14 @@
 * Use `/healthz` endpoint for Kafka Exporter health checks
 * Update Jackson Library to 2.12.6
 * Renew user certificates in User Operator only during maintenance windows
+* Update Cruise Control to 2.5.89
+* Remove TLS sidecar from Cruise Control pod. Cruise Control is now configured to not using ZooKeeper, so the TLS sidecar is not needed anymore.
+
+#### Deprecation of Cruise Control TLS sidecar configuration
+
+Since the Cruise Control TLS sidecar has been removed, the related configuration options in the Kafka custom resource are now deprecated:
+* `.spec.cruiseControl.tlsSidecar`
+* `.spec.cruiseControl.template.tlsSidecar`
```

Comment:
Maybe you can do it as in the previous versions and have section `### Changes, deprecations and removals` and add it under it?